### ISSUE TITLE
how to train with my dataset

### ISSUE THREAD

**Issue by lethanhloiHD** [experiencor/basic-yolo-keras#128]
Hi. I want to train with my own dataset.( with full_yolo )
I want to use my dataset ( example, learn only three classes of people, dog, cat  or the COCO dataset )
What should I do?

AND why do you use the skip_connection in the model after layer 13 ?

THANK YOU !

**Comment by experiencor** [comment]
@lethanhloiHD If you only want to detect people, dog, and cat, you can just put those 3 labels in the ```labels``` setting in the ```config.json```. All other labels will automatically be discarded during model training.

**Comment by thorstenwagner** [comment]
Hi @experiencor,

I'm also wondering about the skip connection, as I cannot find a skip connection in the original yolo papers. 

Best,
Thorsten

**Comment by experiencor** [comment]
@thorstenwagner Skip connection was introduced in the second version (YOLO9000: Better, Faster, Stronger, Section Fine-Grained Features.)

**Comment by RoshanADK** [comment]
Hi @experiencor Thanks a lot for putting up this one. Would you mind please sharing some details regarding the CNN layers in the notebook, just wanted to learn the details :100: Thanks.

Also how can we change the color for each detected class ?
Thanks.

**Comment by experiencor** [comment]
You can refer to the papers for the corresponding CNN:

1. YOLO => https://arxiv.org/abs/1612.08242
2. ResNet => https://arxiv.org/abs/1512.03385
3. InceptionV3 => https://arxiv.org/abs/1512.00567
4. VGG => http://www.robots.ox.ac.uk/~vgg/research/very_deep/
